frontend/src/concepts/hardwareProfiles/useHardwareProfileConfig.ts

```diff
@@ -105,6 +105,7 @@ export const useHardwareProfileConfig = (
   nodeSelector?: NodeSelector,
   visibleIn?: HardwareProfileFeatureVisibility[],
   namespace?: string,
+  hardwareProfileNamespace?: string | null,
```

Comment:
I see now we are using annotation, doesnt the `hardwareProfileNamespace` and namespace give same value? can't we now replace namespace with `hardwareProfileNamespace`?
https://github.com/opendatahub-io/odh-dashboard/pull/4203/files#diff-0e61ca9f1a1c0dd011d4c2e693c5ced1687b2f220aeed574ad3787a7cb388e79R98-R130

Reply:
yes, but when we update the deployment from project-scoped hardware profile to global-scoped hardware profile, then the profile will be selected from project-scoped list and not from global.